Adding pod-qos.md in french (#40591)
kubernetes/website: content/fr/docs/concepts/workloads/pods/pod-qos.md

---
titre: Classes de qualité de service des pods
content_type: concept
poids: 85
---

> Cette page présente les classes de qualité de service (QoS) dans Kubernetes et explique comment Kubernetes affecte une classe QoS à chaque Pods en fonction des contraintes de ressources que vous spécifiez pour les conteneurs dans ce Pod. Kubernetes s’appuie sur cette classification pour prendre des décisions sur les Pods à supprimer lorsqu’il n’y a pas assez de ressources disponibles sur un nœud.

Reviewer: ```suggestion
Cette page présente les classes de Quality of Service (QoS, "Qualité de service") dans Kubernetes et explique comment Kubernetes affecte une classe à chaque Pods en fonction des contraintes de ressources que vous spécifiez pour les conteneurs dans ce Pod. Kubernetes s’appuie sur cette classification pour prendre des décisions sur les Pods à supprimer lorsqu’il n’y a pas assez de ressources disponibles sur un nœud.
```

> ## Cours sur la qualité de service

Reviewer: ```suggestion
## Explication du fonctionnement de qualité de service
```

> Kubernetes classe les Pods que vous exécutez et affecte chaque Pod dans une classe de qualité de service (QoS). Kubernetes utilise cette classification pour influencer les différents pools sont manipulées. Kubernetes effectue cette classification en fonction de la [demandes de ressources](/docs/concepts/configuration/manage-resources-containers/) du {{< glossary_tooltip text="Containers » term_id="container » >}} dans ce Pod, avec le lien entre ces demandes et les limites de ressources. C’est ce qu’on appelle {{< glossary_tooltip text="Quality of Service » term_id="qos-class » >}}(QoS) class. Kubernetes attribue à chaque Pod une classe QoS en fonction des demandes de ressources et les limites de ses conteneurs constitutifs. Les classes QoS sont utilisées par Kubernetes pour décider quels Pods expulser d’un nœud; [Pression du nœud](/docs/concepts/scheduling-eviction/node-pressure-eviction/). Les classes QoS sont `Guaranteed`, `Burstable` et `BestEffort`. Lorsqu’un nœud est à court de ressources, Kubernetes supprimera d’abord les pods `BestEffort` exécutés sur ce nœud, suivis de `Burstable` et enfin des Pods `Guaranteed`.

Reviewer: ```suggestion
Kubernetes classe les Pods que vous exécutez et affecte chaque Pod dans une classe de QoS. Kubernetes utilise cette classification pour influencer les différents pools sont manipulées. Kubernetes effectue cette classification en fonction de la [demandes de ressources](/docs/concepts/configuration/manage-resources-containers/) du {{< glossary_tooltip text="Containers » term_id="container » >}} dans ce Pod, avec le lien entre ces demandes et les limites de ressources. C’est ce qu’on appelle {{< glossary_tooltip text="Quality of Service » term_id="qos-class » >}}(QoS) class. Kubernetes attribue à chaque Pod une classe QoS en fonction des demandes de ressources et les limites de ses conteneurs constitutifs. Les classes QoS sont utilisées par Kubernetes pour décider quels Pods expulser d’un nœud; [Pression du nœud](/docs/concepts/scheduling-eviction/node-pressure-eviction/). Les classes QoS sont `Guaranteed`, `Burstable` et `BestEffort`. Lorsqu’un nœud est à court de ressources, Kubernetes supprimera d’abord les pods `BestEffort` exécutés sur ce nœud, suivis de `Burstable` et enfin des Pods `Guaranteed`.
```

> Kubernetes classe les Pods que vous exécutez et affecte chaque Pod dans une classe de qualité de service (QoS). Kubernetes utilise cette classification pour influencer les différents pools sont manipulées. Kubernetes effectue cette classification en fonction de la [demandes de ressources](/docs/concepts/configuration/manage-resources-containers/) du {{< glossary_tooltip text="Containers » term_id="container » >}} dans ce Pod, avec le lien entre ces demandes et les limites de ressources. C’est ce qu’on appelle {{< glossary_tooltip text="Quality of Service » term_id="qos-class » >}}(QoS) class. Kubernetes attribue à chaque Pod une classe QoS en fonction des demandes de ressources et les limites de ses conteneurs constitutifs. Les classes QoS sont utilisées par Kubernetes pour décider quels Pods expulser d’un nœud; [Pression du nœud](/docs/concepts/scheduling-eviction/node-pressure-eviction/). Les classes QoS sont `Guaranteed`, `Burstable` et `BestEffort`. Lorsqu’un nœud est à court de ressources, Kubernetes supprimera d’abord les pods `BestEffort` exécutés sur ce nœud, suivis de `Burstable` et enfin des Pods `Guaranteed`.

Reviewer: ```suggestion
Kubernetes classe les Pods que vous exécutez et affecte chaque Pod dans une classe de QoS. Kubernetes utilise cette classification pour influencer les différents pools sont manipulées. Kubernetes effectue cette classification en fonction de la [demandes de ressources](/docs/concepts/configuration/manage-resources-containers/) du {{< glossary_tooltip text="Containers » term_id="container » >}} dans ce Pod, avec le lien entre ces demandes et les limites de ressources. C’est ce qu’on appelle {{< glossary_tooltip text="Quality of Service » term_id="qos-class » >}}(QoS) class. Kubernetes attribue à chaque Pod une classe QoS en fonction des demandes de ressources et les limites de ses conteneurs constitutifs. Les classes QoS sont utilisées par Kubernetes pour décider quels Pods expulser d’un nœud; [Pression du nœud](/docs/concepts/scheduling-eviction/node-pressure-eviction/). Les classes QoS sont `Guaranteed`, `Burstable` et `BestEffort`. Lorsqu’un nœud est à court de ressources, Kubernetes supprimera d’abord les pods `BestEffort` exécutés sur ce nœud, suivis de `Burstable` et enfin des Pods `Guaranteed`.
```

---
titre: Classes de qualité de service des pods
content_type: concept
poids: 85
---

> Cette page présente les classes de qualité de service (QoS) dans Kubernetes et explique comment Kubernetes affecte une classe QoS à chaque Pods en conséquence des contraintes de ressources que vous spécifiez pour les conteneurs dans ce Pod. Kubernetes s’appuie sur cette classification pour prendre des décisions sur les Pods à supprimer lorsqu’il n’y a pas assez de ressources disponibles sur un nœud.

Reviewer: ```suggestion
Cette page présente les classes de qualité de service (QoS) dans Kubernetes et explique comment Kubernetes affecte une classe QoS à chaque Pods en fonction des contraintes de ressources que vous spécifiez pour les conteneurs dans ce Pod. Kubernetes s’appuie sur cette classification pour prendre des décisions sur les Pods à supprimer lorsqu’il n’y a pas assez de ressources disponibles sur un nœud.
```

## Cours sur la qualité de service

> Kubernetes classe les Pods que vous exécutez et alloue chaque Pod dans un_quality de service (QoS) class_. Kubernetes utilise cette classification pour influencer la différenceLes gousses sont manipulées. Kubernetes effectue cette classification en fonction de la[demandes de ressources](/docs/concepts/configuration/manage-resources-containers/)du {{< glossary_tooltip text="Containers » term_id="container » >}} dans ce Pod, avecle lien entre ces demandes et les limites de ressources.C’est ce qu’on appelle {{< glossary_tooltip text="Quality of Service » term_id="qos-class » >}}(QoS) classe. Kubernetes attribue à chaque Pod une classe QoS en fonction des demandes de ressourceset les limites de ses conteneurs constitutifs. Les classes QoS sont utilisées par Kubernetes pour déciderquels Pods expulser d’un nœud expérimenté;[Pression du nœud](/docs/concepts/scheduling-eviction/node-pressure-eviction/). Le possibleLes classes QoS sont `Guaranteed`, `Burstable` et `BestEffort`. Lorsqu’un nœud est à court de ressources,Kubernetes supprimera d’abord les pods `BestEffort` exécutés sur ce nœud, suivis de 'Burstable' etenfin des Pods 'garantis'

Reviewer: ```suggestion
Kubernetes classe les Pods que vous exécutez et affecte chaque Pod dans une classe de qualité de service (QoS). Kubernetes utilise cette classification pour influencer les différents pools sont manipulées. Kubernetes effectue cette classification en fonction de la [demandes de ressources](/docs/concepts/configuration/manage-resources-containers/) du {{< glossary_tooltip text="Containers » term_id="container » >}} dans ce Pod, avec le lien entre ces demandes et les limites de ressources. C’est ce qu’on appelle {{< glossary_tooltip text="Quality of Service » term_id="qos-class » >}}(QoS) class. Kubernetes attribue à chaque Pod une classe QoS en fonction des demandes de ressources et les limites de ses conteneurs constitutifs. Les classes QoS sont utilisées par Kubernetes pour décider quels Pods expulser d’un nœud; [Pression du nœud](/docs/concepts/scheduling-eviction/node-pressure-eviction/). Les classes QoS sont `Guaranteed`, `Burstable` et `BestEffort`. Lorsqu’un nœud est à court de ressources, Kubernetes supprimera d’abord les pods `BestEffort` exécutés sur ce nœud, suivis de `Burstable` et enfin des Pods `Guaranteed`.
```

> . Lorsque cette expulsion est due à la pression des ressources, seuls les Pods dépassentLes demandes de ressources sont candidates à l’expulsion.

Reviewer: ```suggestion

Lorsque qu'une expulsion est due à la pression des ressources, seuls les Pods sont dépassés. Les demandes de ressources sont candidates à l’expulsion.
```

#### Critères

Un Pod reçoit une classe QoS de « Burstable » si :

- Le Pod ne répond pas aux critères de la classe QoS 'Garantie'.
- Au moins un conteneur dans le Pod a une demande ou une limite de mémoire ou de CPU.

### Meilleur effort

Les pods de la classe QoS « BestEfforts » peuvent utiliser des ressources de nœud qui ne sont pas spécifiquement affectéesaux Pods d’autres classes QoS. Par exemple, si vous avez un nœud avec 16 cœurs de processeur disponibles pour lekubelet, et vous attribuez 4 cœurs CPU à un Pod 'Guaranteed', puis un Pod dans le 'BestEffort'La classe QoS peut essayer d’utiliser n’importe quelle quantité des 12 cœurs de processeur restants.Le kubelet préfère expulser les pods 'BestEffort' si le nœud est soumis à une pression sur les ressources.

#### Critères

> A Pod has a QoS class of `BestEffort` if it doesn't meet the criteria for either `Guaranteed`

Reviewer: Not traduced

Author: Resolved

or `Burstable`. In other words, a Pod is `BestEffort` only if none of the Containers in the Pod have a
memory limit or a memory request, and none of the Containers in the Pod have a
CPU limit or a CPU request.
Containers in a Pod can request other resources (not CPU or memory) and still be classified as
`BestEffort`.

## Certains comportements sont indépendants de la classe QoS {#class-independent-behavior}

Certains comportements sont indépendants de la classe QoS attribuée par Kubernetes. Par exemple:

> - Tout conteneur dépassant une limite de ressources sera tué et redémarré par le kubelet sansaffectant d’autres conteneurs dans ce pod.

Reviewer: ```suggestion
- Tout conteneur dépassant une limite de ressources sera tué et redémarré par Kubelet sans affecter les autres conteneurs dans ce pod.
```

> - Si un conteneur dépasse sa demande de ressources et que le nœud sur lequel il s’exécute fait facepression des ressources, le Pod dans lequel il se trouve devient un candidat pour [expulsion](/docs/concepts/scheduling-eviction/node-pressure-eviction/).Si cela se produit, tous les conteneurs du pod seront résiliés. Kubernetes peut créer unPod de remplacement, généralement sur un nœud différent.

Reviewer: ```suggestion
- Si un conteneur dépasse sa demande de ressources et que le nœud sur lequel il s’exécute fait face a une pression sur ses ressources, le Pod dans lequel il se trouve devient un candidat pour [eviction](/docs/concepts/scheduling-eviction/node-pressure-eviction/). Si cela se produit, tous les conteneurs du pod seront tués. Kubernetes peut créer un Pod de remplacement sur un nœud différent.
```

> - La demande de ressource d’un Pod est égale à la somme des demandes de ressources deses conteneurs de composants et la limite de ressources d’un pod est égale à la somme deles limites de ressources de ses conteneurs composants.

Reviewer: Incomprehensible, reformulate.

Author: I checked with the English version of the same and translating the above text in English is exactly similar to the English version. The translation here seems correct to me but do correct me.

English Version: https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/workloads/pods/pod-qos.md

Reviewer: I look this tonight

> - Le planificateur kube ne tient pas compte de la classe QoS lors de la sélection des Pods à[préempter](/docs/concepts/scheduling-eviction/pod-priority-preemption/#preemption).La préemption peut se produire lorsqu’un cluster ne dispose pas de suffisamment de ressources pour exécuter tous les Podsvous avez défini.

Reviewer: ```suggestion
- Le planificateur kube ne tient pas compte de la classe QoS lors de la sélection des Pods à [préempter](/docs/concepts/scheduling-eviction/pod-priority-preemption/#preemption). La préemption peut se produire lorsqu’un cluster ne dispose pas de suffisamment de ressources pour exécuter tous les Pods que vous avez défini.
```
